dev/packages/beats/logstash-0.0.1/kibana/search/742e45d0-cbdd-11e7-9852-73e0a9df1bb6-ecs.json

```diff
@@ -21,20 +21,15 @@
                             "alias": null,
                             "disabled": false,
                             "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
-                            "key": "event.module",
+                            "key": "query",
                             "negate": false,
-                            "params": {
-                                "query": "logstash",
-                                "type": "phrase"
-                            },
-                            "type": "phrase",
-                            "value": "logstash"
+                            "type": "custom",
+                            "value": "{\"match_phrase_prefix\":{\"event.dataset\":{\"query\":\"logstash.\"}}}"
```

Comment:
Some this seems to be encoded now. Not sure if this is correct? Also there is a `.` at the end of Logstash.

Reply:
There is a similar form already in filebeat's CEF module. Kibana search file (Network Events [Filebeat CEF]): `68202a5c-c8f2-432f-8c08-04fbfacb95c8.json`. If this is wrong, let me know.

Reply:
Hm, this probably means in Kibana in the end it is double encoded. If this works, good with me :-)